swap_meet/electronics.py

```diff
@@ -1,2 +1,17 @@
-class Electronics:
-    pass
+from swap_meet.item import Item
+
+class Electronics(Item):
+    # WAVE 5
+    def __init__(self, id=None, type='Unknown', condition=0):
+        super().__init__(id, condition)
+
+        self.type = type
+
+    def __str__(self):
```

Comment:
👀 This exhibits the same potential for reusing the parent implementation as `Clothing`.